src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -1770,27 +1770,25 @@ namespace SimAirServingZones {
                                 // and the supply air path outlet
                                 if (SupplyAirPath(SupAirPathNum).OutletNode(SupAirPathOutNodeNum) ==
                                     ZoneEquipConfig(CtrlZoneNum).AirDistUnitCool(ZoneInNum).InNode) {
-                                    if (FindNumberInList(CtrlZoneNum, CtrlZoneNumsCool, NumZonesCool) == 0) {
```

Comment:
Back in the old days (before a zone could be served by more than one airloop) this test was relevant, but now if there's a matching node, this block should execute.  "Zones" cooled or heated are really air terminal units now. The matching nodes should be a unique connection in the whole scheme of things. This change looks big, but it's just removing this if and it's ending brace.

Reply:
Change doesn't look so big when you ignore the whitespace changes: https://github.com/NREL/EnergyPlus/pull/7519/files?w=1